bundle/edu.gemini.model.p1.submit/src/main/scala/edu/gemini/model/p1/submit/Post.scala

```diff
@@ -14,8 +14,8 @@ private[submit] object Post {
     con.setRequestProperty("Content-Type", s"""multipart/form-data; boundary="$BOUNDARY"""")
 
     for {
-      pdf1 <- p.meta.attachments.sortBy(_.index).map(_.name).headOption.flatten.toRight(ClientError("The proposal is missing some PDF attachment.")).right
-      pdf2 <- p.meta.attachments.sortBy(_.index).map(_.name).lastOption.flatten.toRight(ClientError("The proposal is missing some PDF attachment.")).right
+      pdf1 <- p.meta.firstAttachment.toRight(ClientError("The proposal is missing some PDF attachment.")).right
+      pdf2 <- p.meta.secondAttachment.toRight(ClientError("The proposal is missing some PDF attachment.")).right
```

Comment:
Can/should we be more specific about which PDF attachment is missing?

Reply:
Yes, good idea